pkg/volume/portworx/portworx_util.go

```diff
@@ -181,13 +186,24 @@ func createDriverClient(hostname string) (*osdclient.Client, error) {
 	}
 }
 
-func (util *PortworxVolumeUtil) getPortworxDriver(volumeHost volume.VolumeHost) (volumeapi.VolumeDriver, error) {
+func (util *PortworxVolumeUtil) getPortworxDriver(volumeHost volume.VolumeHost, localOnly bool) (volumeapi.VolumeDriver, error) {
+	var err error
+	if localOnly {
+		util.portworxClient, err = createDriverClient(volumeHost.GetHostName())
+		if err != nil {
+			return nil, err
+		} else {
+			glog.Infof("Using portworx local service at: %v as api endpoint", volumeHost.GetHostName())
```

Comment:
IMO -  this log message should only appear if verbosity is set to a lower level, not always.